packages/n/nushell/package.yml

```diff
@@ -0,0 +1,19 @@
+name       : nushell
+version    : 0.87.1
+release    : 1
+homepage   : https://www.nushell.sh/
+networking : yes
+source     :
+    - https://github.com/nushell/nushell/archive/refs/tags/0.87.1.tar.gz : 92087ff56c98acb86dc14e9566748c0f470ad5f13277dd62bda878146535fa83
+license    : MIT
+component  : system.utils
+summary    : A new type of shell.
+description: |
+    Nu takes cues from a lot of familiar territory traditional shells like bash, object based shells like PowerShell, gradually typed languages like TypeScript, functional programming, systems programming, and more. But rather than trying to be a jack of all trades, Nu focuses its energy on doing a few things well
+builddeps  :
+    - rust
+    - cargo
+install    : |
+    cargo build --release --workspace
+    # cargo run --release
+    install -Dm00755 ./target/release/nu $installdir/usr/bin/nu
```

Comment:
Taking a look at the Arch Linux package it looks like they build it with several extra features and include several plugin binaries in their package. Are these not useful for us?